docs/en/observability/synthetics-create-test.asciidoc

```diff
@@ -174,6 +177,92 @@ For more details on getting started with the Synthetics Recorder, see <<syntheti
 
 image::images/synthetics-create-test-script-recorder.png[Elastic Synthetics Recorder after recording a journey and clicking Export]
 
+[discrete]
+[[synthetics-request-param]]
+== Make API requests
+
+You can use the `request` parameter to make API requests independently of the browser interactions.
+
+// Link to Playwright docs?
+// https://playwright.dev/docs/test-api-testing#sending-api-requests-from-ui-tests
+
+NOTE: The `request` parameter is not intended to be used for writing pure API tests. Instead, it is a way to support
+writing plain HTTP requests in service of a browser-based test.
+
+Below is an example of how to use the `request` object to communicate with HTTP endpoints.
+The example demonstrates how to retrieve a token from an HTTP endpoint and use it in a subsequent webpage request.
+
+[source,js]
+-----
+import { journey, step, expect } from '@elastic/synthetics';
+
+journey('Retrieve and use HTTP Headers', ({ page, params, request }) => {
+  // We will populate these variable from example HTTP API responses
+  let apiKey;
+  let decodedApiKey;
+
+  // We will use httpbin.org endpoints for demonstration
+  const apiBaseUrl = 'https://httpbin.org';
+
+  // Let's say we expect the following header key:value from an Auth API
+  const exampleAPIHeaderKey = 'x-api-key';
+  const exampleAPIHeaderValue = 'some api key'; // base64 of 'Example-HTTP-Header-Value'
+
+  // /response-headers endpoint returns passed query params in headers
+  step('Retrieve headers from API', async() => {
+    const resp = await request.get(`${apiBaseUrl}/response-headers?${exampleAPIHeaderKey}=${exampleAPIHeaderValue}`);
+    apiKey = resp.headers()[exampleAPIHeaderKey];
+    expect(apiKey).toEqual(exampleAPIHeaderValue);
+  });
+
+  // /base64 endpoint returns the decoded string
+  step('Use API Key in HTTP request', async() => {
+    const resp = await request.get(`${apiBaseUrl}/base64/${apiKey}`);
+    decodedApiKey = await resp.text();
+    expect(decodedApiKey).toEqual('Example-HTTP-Header-Value');
+  });
+
+  // httpbin.org/headers prints all request headers
+  step('Use API Key in page request', async() => {
+    await page.setExtraHTTPHeaders({ [exampleAPIHeaderKey]: decodedApiKey });
+    await page.goto('https://httpbin.org/headers');
+    await page.waitForSelector(`text=${decodedApiKey}`);
+  });
+
+  step('Send headers to API', async () => {
+    const resp = await request.get('https://httpbin.org/headers', { headers: { 'Foo': 'Bar' } });
+    const respJson = await resp.json();
+    expect(respJson.headers['Foo']).toEqual('Bar');
+  });
+});
+-----
+
+You can also use the `request` parameter in inline journeys:
+
+[source,js]
+-----
+// The apiKey here is just a base64 of 'Example-HTTP-Header-Value'
```

Comment:
This script also breaks, with the error below. I think at some point you changed some hard coded values to vars but didn't update that everywhere.

```
Journey: Retrieve and use HTTP Headers
   ✖  Step: 'Retrieve API key' failed (239 ms)
   ---
      stack: |-
        Error: expect(received).toEqual(expected) // deep equality

        Expected: "some api key"
        Received: "Incorrect Base64 data try: SFRUUEJJTiBpcyBhd2Vzb21l"
            at Step.callback (/home/andrewvc/projects/synthetics-demo/todos/synthetics-tests/test.ts:12:24)
```